hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -890,6 +895,33 @@
       available_input_list: []
       available_output_list: []
       available_inout_list: []
+      param_list:
+      [
```

Comment:
for the parameters that are false for both local / expose, would we still supply them through the hjson?

Reply:
Good point @tjaychen . I think for the local parameters it does not really make sense. Or at least I don't see at the moment what the use of such parameters would be by topgen.

For the non-exposed ones, it is different. There is actually the use case when we don't want to expose the parameter to the very top, but inside `top_earlgrey.sv` we want to provide parameter values that are different from the default. This could be for example a different configuration for a second instantiation of the same module, or as we discussed in #2229 we could provide seed values from a top-level package file to local LFSRs.

To allow for this, I appended another commit that causes un-exposed parameters to be listed in `top_earlgrey.gen.hjson ` and printed in `top_earlgrey.sv` (with the default value specified in the IP hjson). But they are not propagated to the next higher level.

Reply:
sounds great! thanks!